server/service/src/sync/init_programs_data.rs

```diff
@@ -457,6 +458,48 @@ fn encounter_hiv_care_5() -> hiv_care_encounter::HivcareEncounter {
     })
 }
 
+pub fn insert_programs_permissions(connection: &StorageConnection, user_id: String) {
+    UserPermissionRowRepository::new(&connection)
+        .upsert_one(&UserPermissionRow {
+            id: uuid(),
+            user_id: user_id.clone(),
+            store_id: Some("D77F67339BF8400886D009178F4962E1".to_string()),
```

Comment:
the hard coded store id is an issue for me. With my configuration I cannot actually switch to this store, so I cannot access anything 😢 

I used a postgres script to give myself permission for every store which the user has access to ( from `user_store_join` ).